docs/fides/docs/tutorial/annotate.md

```diff
@@ -0,0 +1,266 @@
+    system_type: Application
+    privacy_declarations:
+      - name: Provide e-commerce operations to example customers
+        data_categories:
+          - user.provided.identifiable
+          - user.derived.identifiable
+          - system.operations
+        data_use: provide.system.operations
+        data_subjects:
+          - customer
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+        dataset_references:
+          - flaskr_postgres_dataset
+```
+
+The system is comprised of Privacy Declarations. These can be read colloquially as "This system uses sensitive data types of `data_categories` for `data_subjects` with the purpose of `data_use` at a deidentification level of `data_qualifier`".
+
+In a production app, create as many systems as are necessary to cover all relevant business applications.
+
+### Understanding Systems
+
+In Fidesctl, Systems are used to model things that process data for organizations (applications, services, 3<sup>rd</sup> party APIs, etc.) and describe how these datasets are used for business functions. These groupings are not mutually exclusive; they answer "_How and why are these datasets being used?_" The System resource groups the lowest level of data (your datasets) with your business use cases, and associates qualitative attributes describing the type of data being used.
+
+Systems use the following attributes:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| data_categories | List[FidesKey] | The types of sensitive data as defined by the taxonomy |
+| data_subjects | List[FidesKey] | The individual persons whose data resides in your datasets |
+| data_use | List[FidesKey] | The various categories of data processing and operations within your organization |
+| data_qualifier | List[FidesKey] | The level of deidentification for the dataset |
+| dataset_refereneces | List[FidesKey] | The `fides_key`(s) of the dataset fields used in this Privacy Declaration |
+
+### Maintaining a System Resource
+
+As use cases evolve, your systems' data subjects, data categories, and data uses will change as well. We recommend that updating this resource file becomes a regular part of the development planning process when building a new feature.
+
+---
+
+**PRO TIP**
+
+As more systems are added to a data ecosystem, consider grouping systems into another Fides resource type, called a [Registry](../language/resources.md#registry).
+
+---
+
+## Annotate the Policy
+
+Fidesctl's privacy declarations provide rich metadata about systems, the data categories they process, and the uses of that data. Policies allow you to enforce constraints on these declarations and decide what combinations to allow or reject at your company, thus providing a layer of automation to control data privacy at the source.
+
+Define a single Policy by creating a `flaskr_policy.yml` file in the `fides_resources` directory. This file should contain the following configuration:
```

Comment:
Similar comment - why this policy?

For my quick $0.02, my justification was something like:
> This demo application is quite poorly built with no real controls on user data, so the Fides policy for it is relatively restrictive. First, we state that we shouldn't use identifiable data for *anything* other than the primary use for the app, which is to provide the system (after all, it's just a demo app!).
> Secondly, I like to add a defensive policy to reject collecting any sensitive data at all, as a safe default. This is the type of policy you might add to all my projects, and then you can make exceptions to this policy if you are working on a project that requires these categories.